chia/cmds/wallet.py

```diff
@@ -1139,12 +1139,14 @@ def nft_transfer_cmd(
 )
 @click.option("-f", "--fingerprint", help="Set the fingerprint to specify which key to use", type=int)
 @click.option("-i", "--id", help="Id of the NFT wallet to use", type=int, required=True)
-def nft_list_cmd(wallet_rpc_port: Optional[int], fingerprint: int, id: int) -> None:
+@click.option("--num", help="Number of NFTs to return", type=int, default=100_000)
+@click.option("--start-index", help="Which starting index to list NFTs from, used in pagination", type=int, default=0)
```

Comment:
there is no pagination (as in, pages are not used but offset), so I'd remove last part of the sentence

Reply:
will do

Reply:
I've rephrased it a little for I think a little better legibility, after removing mention of pagination https://github.com/Chia-Network/chia-blockchain/pull/15342/commits/17ea29321cd1b5239c39e33eaff7b1a10d272169